### DISCUSSION TITLE
Back and Forward navigation in CACHE tab

### DISCUSSION THREAD

**Discussion by joeshub** [apollographql/apollo-client-devtools#1034] [Ideas]
Navigation in the `CACHE` tab would be so much more productive if there was a simple way (UX based or Keyboard based) way to go back and forward. Currently, if you start navigating the cache from the `ROOT_QUERY` and want to discover another leaf node somewhere in your path, you have to start the navigation process all the way from the top all over again.

**Comment by jerelmiller** [comment]
Hey @joeshub 👋 

Just wanted to let you know that I put up a PR to add this functionality in https://github.com/apollographql/apollo-client-devtools/pull/1394. I've wanted/needed this feature myself in some of my own projects lately so I went ahead and implemented it. I'm hoping to get this released within the next week or so so stay tuned!

**Comment by jerelmiller** [comment]
This has now been released in [4.15.0](https://github.com/apollographql/apollo-client-devtools/releases/tag/v4.15.0) and should be live in the extension stores within the next couple hours. Hope you enjoy the new feature!
